src/vaults/script/distribute.ts

```diff
@@ -65,13 +70,63 @@ const distribute = async (
   }
   if (mergeTrees) {
     const mergedTree = mergeMerkleTrees(trees);
-    const epoch = getAllProofs()[0].epoch;
+
+    const lastProof = getAllProofs()[0];
+    const epoch = lastProof.epoch;
+
     const filename = `${baseDir}/${epoch}-merged.json`;
     console.log(`Saving proof for ${filename}`);
+
     await fs.promises.writeFile(
       filename,
       JSON.stringify({ epoch, vaults: vaults.map((v) => v.address), ...mergedTree }, null, 2)
     );
+    if (createBatch) {
```

Comment:
Note that we'd need to update it in the next uprgade

Reply:
Why? the script is claiming, transferring tokens to the rewards distributor and updating root, witch will be the same after the upgrade